src/test/java/uk/gov/pay/adminusers/fixtures/EventFixtureUtil.java

```diff
@@ -0,0 +1,101 @@
+package uk.gov.pay.adminusers.fixtures;
+
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.model.SendMessageResult;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import org.apache.commons.lang3.NotImplementedException;
+import uk.gov.pay.adminusers.infra.SqsTestDocker;
+
+public class EventFixtureUtil {
+
+    public static String insert(AmazonSQS sqsClient, String eventType, String serviceId, Boolean live, String resourceExternalId,
+                                String parentResourceExternalId, String eventData) {
+        String messageBody = String.format("{" +
+                        "\"resource_external_id\": \"%s\"," +
+                        "\"service_id\": \"%s\"," +
+                        "\"live\": \"%s\"," +
+                        (parentResourceExternalId == null || parentResourceExternalId.isEmpty() ? "%s" : "\"parent_resource_external_id\": \"%s\",") +
+                        "\"event_type\":\"%s\"," +
+                        "\"event_details\": %s" +
+                        "}",
+                resourceExternalId,
+                serviceId,
+                live,
+                parentResourceExternalId == null ? "" : parentResourceExternalId,
+                eventType,
+                eventData
+        );
+
+        SendMessageResult result = sqsClient.sendMessage(SqsTestDocker.getQueueUrl("event-queue"), messageBody);
+        return result.getMessageId();
+    }
+
+    public static PactDslJsonBody getAsPact(String serviceId, Boolean live, String eventType, String resourceExternalId,
+                                            String parentResourceExternalId, JsonNode eventData) {
+        PactDslJsonBody message = new PactDslJsonBody();
+
+        message.stringType("event_type", eventType);
+        message.stringType("resource_external_id", resourceExternalId);
+        message.booleanType("live", live);
+        if (parentResourceExternalId != null && !parentResourceExternalId.isEmpty()) {
+            message.stringType("parent_resource_external_id", parentResourceExternalId);
+        }
+        if (serviceId != null) {
+            message.stringType("service_id", serviceId);
+        }
+        if (live != null) {
+            message.booleanType("live", live);
+        }
+
+        PactDslJsonBody eventDetailsPact = getNestedPact(eventData);
+
+        message.object("event_details", eventDetailsPact);
+        PactDslJsonBody snsMessage = new PactDslJsonBody();
+        snsMessage.object("Message", message);
```

Comment:
What's the point of this section? `snsMessage` isn't being used in the return.

Reply:
Good spot - that's from when I was fiddling around with how to construct the message when I was trying to get it working and should be removed